x-pack/filebeat/module/zeek/connection/ingest/pipeline.json

```diff
@@ -0,0 +1,54 @@
+{
+  "description": "Pipeline for normalizing Zeek conn.log",
+  "processors": [
+    {
+      "script": {
+        "lang": "painless",
+        "source": "ctx.event.created = (long)ctx.event.created * params.multiplier",
```

Comment:
Since there's a multiplication, I assume you have a unix ts here? `event.created` should be an ISO8601 timestamp, not an integer.

Reply:
The original ts field is a long in the units of seconds. It is converted to ms for the date typed field.